src/Form/matching-types.d.ts

```diff
@@ -104,24 +104,28 @@ type SupportedSubTypes =
     | 'creditCards.expirationYear'
     | 'creditCards.expiration'
 
+type SupportedSubTypesOrString =
+    | SupportedSubTypes
+    | string
+
 interface InputTypeConfigBase {
     type: SupportedMainTypes,
     getIconFilled: (input: HTMLInputElement, form: Form) => string,
     getIconBase: (input: HTMLInputElement, form: Form) => string,
     shouldDecorate: (input: HTMLInputElement, form: Form) => boolean,
     dataType: 'Addresses' | 'Credentials' | 'CreditCards' | 'Identities' | '',
-    displayTitlePropName: (input: HTMLInputElement, data: any) => string
+    displayTitlePropName: (input: SupportedSubtypesOrString, data: any) => string
```

Comment:
Should we rename all these `input` to `_subtype`?

Reply:
Yes, sorry missed that in the rebase :)